### DISCUSSION TITLE
Wrong instructions for setup with Gatsby

### DISCUSSION THREAD

**Discussion by Manc** [mantinedev/mantine#341] [Feedback]
I was just about to try out Mantine and Gatsby and ran into an issue on https://mantine.dev/getting-started/ selecting Gatsby.

The init command given is the one for Next.js:

    npx create-next-app --ts

I think I'll manage to work around it. :) Just wanted to let you know. It may lead to some confusion for others.

**Comment by rtivital** [comment]
Thanks for reporting, we already fixed this issue, it will be correct when version 3.1 is out
